### ISSUE TITLE
Future-proof against changes to `git read-tree --prefix`

### ISSUE THREAD

**Issue by rellen** [asdf-vm/asdf-plugin-template#53]
**Is your feature request related to a problem? Please describe.**

Recent versions of git will disallow `git read-tree --prefix=/ ...`
See: https://www.spinics.net/lists/git/msg433678.html
The template will fail if using a recent version of git.
Tested on `git version 2.36.0`

```
Already on 'template'
Preparing worktree (detached HEAD 843b4d4)
HEAD is now at 843b4d4 Initial commit
Switched to a new branch 'out'
fatal: Invalid prefix, prefix cannot start with '/'
```

**Describe the solution you'd like**

`git read-tree --prefix="" ...` seems to work for `setup.bash` changing the below line.
https://github.com/asdf-vm/asdf-plugin-template/blob/main/setup.bash#L170

I am not sure if this is the most robust solution.

Thanks for providing this template!

**Comment by jthegedus** [comment]
Thanks for the report, makes sense to me. Are you able to create a PR?

**Comment by rellen** [comment]
@jthegedus PR created :)

**Comment by jthegedus** [comment]
closed in #54
